influxdb3_py_api/src/system_py.rs

```diff
@@ -256,6 +278,20 @@ impl PyPluginCallApi {
     }
 }
 
+impl PyPluginCallApi {
+    fn write_to_logger(&self, level: LogLevel, log_line: String) {
+        if let Some(logger) = &self.logger {
+            let processing_engine_log = ProcessingEngineLog::new(
+                logger.sys_event_store.time_provider().now(),
+                level,
+                logger.trigger_name.clone(),
```

Comment:
Can this be changed to `Arc<str>`? Would prefer to not be cloning a bunch of repeated strings.